src/App.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import LinkComponent from './components/LinkComponent';
+// import Data from './Visualization/EmptyData';
+// import Visualization from './Visualization/Visualization';
```

Comment:
Since you are using the Visualization component inside the LinkComponent, it is safe to remove these two from here.

Reply:
Sure I'll remove those!